docs/fomega/src/AlgTypes.hs

```diff
@@ -1,167 +0,0 @@
-{-# LANGUAGE GADTs           #-}
```

Comment:
Why are you deleting this file?

Reply:
I must have updated my fork before the most recent branch of the IOHK repo. :(